bakery/codec.go

```diff
@@ -85,143 +91,218 @@ func encodeCaveatIdV1(
 	return buf, nil
 }
 
+// encodeCaveatV2 creates a version 2 third-party caveat.
+func encodeCaveatV2(
```

Comment:
given that these functions are internal is there much value in having them with one-line implementations?